src/web/tools/GestureHandlerOrchestrator.ts

```diff
@@ -67,6 +67,17 @@ export default class GestureHandlerOrchestrator {
   }
 
   private tryActivate(handler: GestureHandler): void {
+    if (
+      this.gestureHandlers.some(
+        (otherHandler) =>
+          this.shouldHandlerWaitForOther(handler, otherHandler) &&
+          otherHandler.getState() === State.END
+      )
```

Comment:
Can you extract this to a helper method? Seeing something like `if (shouldBeCancelledByAlreadyFinishedHandler...)` is much more readable than this (name itself is questionable, but you get the point).

Reply:
Done in [2375a47](https://github.com/software-mansion/react-native-gesture-handler/pull/2759/commits/2375a475d759eee7b7e5a99a0983fbd83df02fe3)